js/bundle.go

```diff
@@ -451,6 +451,16 @@ func (b *Bundle) setInitGlobals(rt *sobek.Runtime, vu *moduleVUImpl, modSys *mod
 	}
 	warnAboutModuleMixing("module")
 	warnAboutModuleMixing("exports")
+
+	rt.SetFinalImportMeta(func(o *sobek.Object, mr sobek.ModuleRecord) {
+		_ = o.Set("resolve", func(specifier string) (string, error) {
```

Comment:
Shouldn't we do something with the error in case there's any? If it isn't supposed to happen, at all, I'm generally in favor of panicking.